.github/workflows/build.yml

```diff
@@ -0,0 +1,29 @@
+name: Build Eleventy
```

Comment:
I've verified this workflow in my fork of this repository. Upon merging to main, the site it built, then deployed.